phoenix-core/src/it/java/org/apache/phoenix/query/MetaDataCachingIT.java

```diff
@@ -142,18 +146,25 @@ public void testSystemTablesAreInCache() throws Exception {
      */
     @Test
     public void testGlobalClientCacheMetrics() throws Exception {
-        int numThreads = 5;
+        int numThreads = 1;
```

Comment:
why this change?

Reply:
not sure I remember this one, I will revert this.